branca/_version.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I don't think we should put this file through our styling things, since it is auto-generated by Versioneer.